### PR TITLE
Csv parser

### PR DESCRIPTION
**Pull request by ArnoutvdKnaap**
added support for csv files.

### CODE REVIEW

File: src/test/java/context/healthinformatics/Parser/XMLParserTest.java [hunk 1]
```diff
@@ -35,22 +35,27 @@ public void before() {
 	 */
 	@Test
 	public void correctTest() {
+		final int startLine = 7;
+		final int col1 = 2;
+		final int col2 = 5;
+		final int col3 = 6;
```
Comment (jelmerdeboer) [inline src/test/java/context/healthinformatics/Parser/XMLParserTest.java]: Ik zou die variabele voor de hele test beschikbaar maken ipv lokaal. Kan je het altijd eventueel hergebruikem

**Comment by jelmerdeboer** [comment]
JavaDoc etc is ok. When my comments are answered or resolved, its ok for me

File: src/test/java/context/healthinformatics/Parser/XMLParserTest.java (hunk 1 above)
Comment (ArnoutvdKnaap) [inline src/test/java/context/healthinformatics/Parser/XMLParserTest.java]: That doesnt make any sense. We would like  to be able to test different files/ settings per test. So global would be bad.

**Comment by pascalremeijsen** [comment]
Reviewed, and looked good. No comments.

**Pull request merged**